core-war/src/main/java/org/silverpeas/web/attachment/DragAndDrop.java

```diff
@@ -81,9 +87,8 @@ public void doPost(HttpServletRequest req, HttpServletResponse res)
 
     final String versionType = req.getParameter("Type");
 
-
     final HttpRequest request = HttpRequest.decorate(req);
-    request.setCharacterEncoding(CharEncoding.UTF_8);
+    request.setCharacterEncoding(Charsets.UTF_8.name());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a "try/catch" block for "setCharacterEncoding". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1989)